test/e2e/testdata/vpa/crd.yaml

```diff
@@ -1,4 +1,4 @@
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This needs to be updated further with schema changes to be a valid v1 CRD, see https://github.com/operator-framework/operator-lifecycle-manager/pull/2228/files#diff-c3d03b07312e2fc84b0bd43696d3b855b171253a3f1e48dc58b7b131595014dc

In general for most of these tests simply bumping the version won't be enough, they won't be valid v1 CRDs without some sort of schema validation. I have been using dummy schemas in my PR and they seem to work fine. Things like https://github.com/operator-framework/operator-lifecycle-manager/pull/2228/files#diff-ba504d941f6b7c7c6d826ab787f8531904b4bc5d1edb0658a28a8dbb04ff17a8R296